modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -247,4 +247,25 @@ definition {
 		}
 	}
 
+	@description = "LPS-155304. Assert filter is preloaded when entering on a FDS page for first time."
+	@priority = "5"
+	test WillRenderAsInitialState {
+		task ("Then a filter is preloaded") {
+			VerifyElementPresent(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+		}
```

Comment:
I think this is an awkward verify/assert since it basically duplicates the last assertion. I would probably remove this completely, and replace with a When entering on a page which uses FDS and have the step click on a specific tab on the sample portlet to help make sure the test makes assertions on our assumed tab/dataset.

Reply:
Agree, I didn't know how to do it without editing the steps, but it makes so much sense and I remove the step and replaced it as you suggested.
Thanks!